src/long.ts

```diff
@@ -454,7 +454,7 @@ export class Long extends BSONValue {
     let unsigned = false;
     if (typeof unsignedOrRadix === 'number') {
       // For goog.math.long compatibility
-      ((radix = unsignedOrRadix), (unsignedOrRadix = false));
+      (radix = unsignedOrRadix), (unsignedOrRadix = false);
```

Comment:
`npm run format` did this, don't really care about keeping or dropping this part